platform/platform-app/src/main/resources/features.xml

```diff
@@ -929,10 +929,10 @@
         <bundle>mvn:ddf.mime.core/mime-core-configurableresolver/${project.version}</bundle>
         <bundle>mvn:org.apache.tika/tika-core/${tika.version}</bundle>
         <bundle>mvn:org.codice.thirdparty/tika-bundle/${tika.thirdparty.bundle.version}</bundle>
-        <configfile finalname="/etc/DDF_Custom_Mime_Type_Resolver-csw.config">mvn:ddf.platform/platform-app/${project.version}/config/csw</configfile>
-        <configfile finalname="/etc/DDF_Custom_Mime_Type_Resolver-geojson.config">mvn:ddf.platform/platform-app/${project.version}/config/geojson</configfile>
-        <configfile finalname="/etc/DDF_Custom_Mime_Type_Resolver-nitf.config">mvn:ddf.platform/platform-app/${project.version}/config/nitf</configfile>
-        <configfile finalname="/etc/DDF_Custom_Mime_Type_Resolver-xml.config">mvn:ddf.platform/platform-app/${project.version}/config/xml</configfile>
+        <configfile finalname="${karaf.etc}/DDF_Custom_Mime_Type_Resolver-csw.config">mvn:ddf.platform/platform-app/${project.version}/config/csw</configfile>
+        <configfile finalname="${karaf.etc}/DDF_Custom_Mime_Type_Resolver-geojson.config">mvn:ddf.platform/platform-app/${project.version}/config/geojson</configfile>
+        <configfile finalname="${karaf.etc}/DDF_Custom_Mime_Type_Resolver-nitf.config">mvn:ddf.platform/platform-app/${project.version}/config/nitf</configfile>
+        <configfile finalname="${karaf.etc}/DDF_Custom_Mime_Type_Resolver-xml.config">mvn:ddf.platform/platform-app/${project.version}/config/xml</configfile>
```

Comment:
I see a few more finalnames in this file that should be updated too: https://github.com/codice/ddf/blob/c8ad93d5455834983383d1f2a3cbef9ee5dffac5/platform/platform-app/src/main/resources/features.xml#L988,L991,L1006,L1009,L1012,L1015,L1018,L1021,L1024